File: freq_mon.py
from datetime import date,timedelta

type_bot = "xdotool key "
year = 2022

month=int(input("Mês: "))

weekdays="STQQSSD"
mday=date(year, month, 1)
while mday.month == month:
    print(type_bot, weekdays[mday.weekday()])
    print(type_bot, "Tab")
    mday=mday+timedelta(days=1)

File: freq_mon.js
const year=2023;
const month=prompt("Mês (0=Janeiro):");
const weekdays="DSTQQSS";

var mday=new Date(year,month,1);
var cabe=[];
while (mday.getMonth() === Number(month)) {
    cabe.push(weekdays[mday.getDay()]);
    mday.setDate(mday.getDate()+1);
}
console.log(cabe)
